### DISCUSSION TITLE
unable to create a new branch

### DISCUSSION THREAD

**Discussion by roaa-art** [skills/.github#430] [Introduction to GitHub]
<img width="662" alt="creatNewBranch" src="https://github.com/skills/.github/assets/143911798/3fe2f0c4-41a0-4541-af04-b6c6a81fd80a">
hello guys
I am not getting an option to create a new branch, could you please help me with this?

**Comment by cmwilson21** [comment]
Hey @roaa-art! It looks like you are trying to make these changes to our public template. You can follow the directions under "[How to start this course](https://github.com/skills/introduction-to-github#how-to-start-this-course)" to copy the template and create your own repo where you can make these changes 👍
